plugins/pulp_docker/plugins/importers/importer.py

```diff
@@ -313,6 +321,27 @@ def _import_manifest(conduit, unit, dest_repo):
         return units_added
 
     @staticmethod
+    def _import_manifest_list(conduit, unit, dest_repo):
+        units_added = set()
+
+        # Add manifest lists  and catalog referenced manifests
+        manifest_digests = set()
+        repository.associate_single_unit(dest_repo, unit)
```

Comment:
What do you think about doing this at the end of the function, after the manifests have been imported? Then in case it stopped part-way through for some reason, we won't get stuck with a manifest list in a repo that doesn't have all of its manifests.